templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Post's List:</h1>
+  <ul>
+    {% for post in post_list%}
+      <li><a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a>
+        <br>
+        {{ post.content }} <br>
+        <br>{{ post.owner.first_name }} {{ post.owner.last_name }} <br> {{ post.created_time }}
+        <br>
+         <p>Comments: {{ post.commentary_count }}</p>
```

Comment:
Make sure that 'commentary_count' is a valid attribute of the 'post' object. If it's a method or a property, ensure it's correctly implemented in the model.